docs/source/whatsnew_1_2.md

```diff
@@ -0,0 +1,59 @@
+with options to specify the number of epochs. Users can access APIs for training, dataset conversion, data preprocessing, and other components.
+Please check out [the tutorials](https://github.com/Project-MONAI/tutorials/tree/main/nnunet) for more details.
+
+## TensorRT optimized networks
+[NVIDIA TensorRT](https://developer.nvidia.com/tensorrt) is an SDK for high-performance deep learning inference,
+includes a deep learning inference optimizer and runtime that delivers low latency and high throughput for inference applications.
+It can accelerate the deep learning model forward computation on the NVIDIA GPU.
+In this release, the `trt_export` API to export the TensorRT engine-based TorchsScript model has been integrated into the MONAI bundle.
+Users can try to export bundles with it. A few bundles in the MONAI model zoo,
+like the [spleen_ct_segmentation](https://github.com/Project-MONAI/model-zoo/tree/dev/models/spleen_ct_segmentation)
+and [endoscopic_tool_segmentation](https://github.com/Project-MONAI/model-zoo/tree/dev/models/endoscopic_tool_segmentation) bundles,
+have already been exported and benchmarked. For more details about how to export and benchmark a model,
+please go to this [tutorial](https://github.com/Project-MONAI/tutorials/blob/main/acceleration/TensorRT_inference_acceleration.ipynb).
+
+
+## MetricsReloaded integration
+MetricsReloaded - a new recommendation framework for biomedical image analysis validation - is released publicly
+via https://github.com/Project-MONAI/MetricsReloaded. Binary and categorical metrics computing modules are included in this release,
+using MetricsReloaded as the backend. [Example scripts](https://github.com/Project-MONAI/tutorials/tree/main/modules/metrics_reloaded) are made available to demonstrate the usage.
+
+
+## Bundle workflow APIs
+`BundleWorkflow` abstracts the typical workflows (such as training, evaluation, and inference) with three main interfaces:
+`initialize`, `run`, and `finalize`, applications use these APIs to execute a bundle.
+It unifies the required properties and optional properties for the workflows, downstream applications
+can invoke the methods instead of parsing configs with keys.
+In this release, `ConfigWorkflow` class is also created for JSON and YAML config-based bundle workflows for improved pythonic usability.
+
+
+## Modular patch inference
+In patch inference, patches are extracted from the image, the inference is run on those patches, and outputs are merged
+to construct the result image corresponding to the input image. Although depending on the task, model, and computational/memory resources,
+the exact implementations of a patch inference may vary, the overall process of splitting, running inference and merging the results remains the same.
+In this release, we have created a modular design for patch inference, where it defines the overall process while abstracts away the specific
```

Comment:
Two minor suggestions:

- `where it` -> `which`
- `while abstracts` -> `while abstracting`